x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/cytoscape_options.tsx

```diff
@@ -33,6 +34,9 @@ function shapeForNode(el: cytoscape.NodeSingular, theme: EuiThemeType): MapShape
       return MAP_SHAPES.DIAMOND;
     case JOB_MAP_NODE_TYPES.TRAINED_MODEL:
       return MAP_SHAPES.TRIANGLE;
+    case JOB_MAP_NODE_TYPES.INGEST_PIPELINE:
```

Comment:
Can we have an extra item added to the legend for this new node type?

Reply:
Also we add a fill color for the source node, which isn't being applied when the source node is a non-DFA trained model. Can the fill be applied for all types of trained models?

<img width="386" alt="image" src="https://github.com/elastic/kibana/assets/7405507/43396140-ad79-452e-b322-70451d5e13c0">

Reply:
Updated here [`d1ab7f6` (#162443)](https://github.com/elastic/kibana/pull/162443/commits/d1ab7f68674e03b49a3aadd99a95e96ac65d7368)
<img width="1267" alt="Screen Shot 2023-07-27 at 13 26 22" src="https://github.com/elastic/kibana/assets/43350163/e166a751-6baa-4ebb-b229-2077857455ca">